code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -171,6 +171,15 @@
 	if(!src.can_close())
 		return 0
 
+	for(var/atom/movable/AM in src.loc)
```

Comment:
You're only looping through carbons so `for (var/mob/living/carbon/C in src.loc)` would be clearer

Reply:
Good idea.